ModelCatalogueCorePluginTestApp/grails-app/assets/javascripts/modelcatalogue/core/ui/bs/navigationRightActions.coffee

```diff
@@ -355,7 +355,7 @@ angular.module('mc.core.ui.bs.navigationRightActions', ['mc.util.ui.actions', 'm
       label: "Import MC"
       icon:  'fa fa-upload fa-fw'
       action: ->
-        messages.prompt('Import Model Catalogue DSL File', '', type: 'new-mc-import')
```

Comment:
If you take a look at the issue we agreed that the name will be "Import Model Catalogue DSL File" in both situations so instead of changing this line you'd rather change the `Title` above